var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -33,17 +33,10 @@ class Openblas(MakefilePackage):
     version('0.2.16', sha256='766f350d0a4be614812d535cead8c816fc3ad3b9afcd93167ea5e4df9d61869b')
     version('0.2.15', sha256='73c40ace5978282224e5e122a41c8388c5a19e65a6f2329c2b7c0b61bacc9044')
 
-    variant(
-        'shared',
-        default=True,
-        description='Build shared libraries as well as static libs.'
-    )
-    variant('ilp64', default=False, description='64 bit integers')
+    variant('ilp64', default=False, description='Force 64-bit Fortran native integers')
     variant('pic', default=True, description='Build position independent code')
-
-    variant('cpu_target', default='auto',
-            description='Set CPU target architecture (leave empty for '
-                        'autodetection; GENERIC, SSE_GENERIC, NEHALEM, ...)')
+    variant('shared', default=True, description='Build shared libraries')
+    variant('static', default=True, description='Build static libraries')
```

Comment:
Having separate static/shared follows the conventions of other packages... mfem, sstmacro, sundials, to name a few. For now I advocate staying consistent with them until there's a decision to unify the usage across packages.

Reply:
Grepping variant names `static` seems to be used 10 times out of which 5 times in packages with also a `shared` variant (`sundials`, `mpileaks`, `libflame`, `mfem`, `sst-macro`). So the few you named are 60% of the cases :slightly_smiling_face: (This is just to say that in general the convention seems to be to avoid `static` as a variant with just a few exceptions that probably slipped through review?)

Reply:
Oh well I guess I just got lucky. I'm fine with removing the `static` variant; I only added it on this MR because I'd seen a couple of those packages and noticed while perusing OpenBLAS' makefile that it was an option.